catkin_tools/config.py

```diff
@@ -35,6 +35,7 @@
 install: config --install
 p: create pkg
 test: build --verbose --make-args test --
+roslint: build -iv --make-args roslint --
```

Comment:
Does the `roslint` target return a non-zero error code when there is a problem? If it does I would argue that there should be neither `-v` nor `-i`. I think those are preferences that the user should select. `-i` is particularly suspect and I'd say it shouldn't be the default even if `-v` is left.